ffi/diplomat/ffi_coverage/src/main.rs

```diff
@@ -275,6 +289,9 @@ lazy_static::lazy_static! {
         // https://github.com/rust-diplomat/diplomat/pull/244
         "icu::locid::Locale::UND",
 
+        // Need to think about how to expose DataErrorKind for this to work
+        "icu_provider_adapters::empty::EmptyDataProvider::new_with_error_kind#FnInStruct",
```

Comment:
@Manishearth  Why does this still show up in missing_apis.txt?

Reply:
The pound sign is unnecessary